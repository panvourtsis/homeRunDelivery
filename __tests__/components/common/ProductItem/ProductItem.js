import 'react-native'
import React from 'react'
import productMock from '@tests/__mocks__/product.json'
import { ProductItem } from '@components/common/index'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <ProductItem product={productMock} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
