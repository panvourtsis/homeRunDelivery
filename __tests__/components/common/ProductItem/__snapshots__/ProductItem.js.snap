// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "padding": 15,
      "width": 150,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "height": 120,
          "marginBottom": 5,
          "width": 120,
        },
        Object {
          "overflow": "hidden",
        },
      ]
    }
  >
    <FastImageView
      onFastImageError={undefined}
      onFastImageLoad={undefined}
      onFastImageLoadEnd={undefined}
      onFastImageLoadStart={undefined}
      onFastImageProgress={undefined}
      resizeMode="contain"
      source={
        Object {
          "priority": "normal",
          "uri": "https://imagine6.convibo.com/product/54836/92313/original.jpeg",
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#000",
        "fontWeight": "bold",
      }
    }
  >
    Â£0.20
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#A1A1A1",
      }
    }
  >
    Bananas Loose
  </Text>
</View>
`;
